### PR TITLE
added 2 files

### PR DESCRIPTION
**Pull request by UniAleks**

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import java.math.BigInteger;
+import java.util.Scanner;
+
+public class Main {
+/*     public static void main(String[] args) {
+          System.out.println("First element");
+      }
+  }
+ */
+    //Задача №1
```
Comment (YeaaaaH) [inline Main.java]: Add separate classes for each task, as mentioned in description

File: Main.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+int Sum = a+b;
+int Min = a-b;
+int Seg = a/b;
+int Mul = a*b;
+int Rem = a%b;
+int Per = a*100/b;
+        System.out.println("Сумма:"+Sum);
+        System.out.println("Разность:"+Min);
+        System.out.println("Произведение:"+Seg);
+        System.out.println("Деление:"+Mul);
```
Comment (YeaaaaH) [inline Main.java]: Replace seg and mul descriptions

File: Main.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+     */
+    //Задача №4
+    /*
+    public static void main(String[] args) {
+        int a = 15;
+        int b = 12;
+        if (a > b)
+            System.out.println(a);
+        else
+            System.out.println(b);
```
Comment (YeaaaaH) [inline Main.java]: what if numbers are equal?

File: Main.java [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+if (c > a && c > b)
+    System.out.println(c);
+    }
+}
+  */
+    //Задача №6
+    public static void main(String[] args) {
+        int a = (int) (Math.random() * 100);
+        int b = (int) (Math.random() * 100);
+        int c = (int) (Math.random() * 100);
```
Comment (YeaaaaH) [inline Main.java]: Math.random() func returns values from 0 included to 1 excluded, zero is not frequent, but still possible

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
please fix comments

File: Task2.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
+public class Task2 {
+    public static void main(String[] args) {
+    int a = 6;
+    int b = 4;
+    int Sum = a+b;
```
Comment (YeaaaaH) [inline Task2.java]: variables have to start from lowercase letter according to ''Naming Conventions''

File: Task7.java [hunk 6]
```diff
@@ -0,0 +1,10 @@
+public class Task7 {
+    public static void main(String[] args) {
+        int a = 1 + (int) (Math.random() * 99 );
+        int b = 1 + (int) (Math.random() * 99 );
+        int c = 1 + (int) (Math.random() * 99 );
```
Comment (YeaaaaH) [inline Task7.java]: You don't need to convert to int, Math.random() returns double, use it instead.
Also you can put a value of random func in output like that:
System.out.println(1 + Math.random()*99);

File: Task3.java [hunk 7]
```diff
@@ -0,0 +1,7 @@
+public class Task3 {
+    public static void main(String[] args) {
+        int Calcium = 12;
+        int Fahrenheit = (Calcium * 9 / 5) + 32;
```
Comment (YeaaaaH) [inline Task3.java]: variables have to start from lowercase letter according to ''Naming Conventions''

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
still need to fix minor issues

**Review by YeaaaaH — APPROVED** [review]

File: Main.java (hunk 1 above)
Comment (UniAleks) [reply Main.java]: fixed

File: Main.java (hunk 2 above)
Comment (UniAleks) [reply Main.java]: fixed

File: Main.java (hunk 3 above)
Comment (UniAleks) [reply Main.java]: fixed

File: Main.java (hunk 4 above)
Comment (UniAleks) [reply Main.java]: fixed

File: Task2.java (hunk 5 above)
Comment (UniAleks) [reply Task2.java]: fixed

File: Task7.java (hunk 6 above)
Comment (UniAleks) [reply Task7.java]: fixed

File: Task3.java (hunk 7 above)
Comment (UniAleks) [reply Task3.java]: fixed

**Pull request merged**